packages/node/src/behaviors/operational-credentials/OperationalCredentialsServer.ts

```diff
@@ -395,7 +402,10 @@ export class OperationalCredentialsServer extends OperationalCredentialsBehavior
     override addTrustedRootCertificate({ rootCaCertificate }: OperationalCredentials.AddTrustedRootCertificateRequest) {
         const failsafeContext = this.#failsafeContext;
 
-        if (failsafeContext.hasRootCert) {
+        // TC_OPCREDS_3_4 fails if we don't allow set of the root certificate in updates, even though that's illegal and
```

Comment:
where exactly in the python test this is set before?

https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_OPCREDS_3_4.py

I only see addTrustedRoot Cert call once?

Reply:
So see above ... we need to debug what happens here and who sets this cert why? Then we can see if it is an issue on our side or in the test or where ever